neqo-transport/src/recovery/mod.rs

```diff
@@ -863,6 +863,15 @@ impl LossRecovery {
         // pto_time to increase which might cause PTO for later packet number spaces to not fire.
         if let Some(pn_space) = pto_space {
             qtrace!([self], "PTO {}, probing {:?}", pn_space, allow_probes);
+            // If we hit a PTO while we don't have a largest_acked yet, also do a congestion control
+            // reaction (because otherwise none would happen).
```

Comment:
```suggestion
            // Packets are only declared as lost, relative to the
            // `largest_acked`. If we hit a PTO while we don't have a
            // largest_acked yet, also do a congestion control reaction (because
            // otherwise none would happen).
```

Is this correct? If so, change above would have made it easier for me to understand.